.travis.yml

```diff
@@ -1,7 +1,6 @@
 
 language: python
 python:
-  - "2.6"
```

Comment:
We need to keep testing 2.6 for as long as we want to support RHEL/CentOS 6.  It doesn't go EOL until 2020-11-30.

Reply:
Rolled this back